Source/WebCore/svg/properties/SVGPropertyOwnerRegistry.h

```diff
@@ -363,7 +363,7 @@ class SVGPropertyOwnerRegistry : public SVGPropertyRegistry {
         return lookupRecursivelyAndApplyBaseTypes<Functor, I + 1>(attributeName, functor);
     }
 
-    OwnerType& m_owner;
+    Ref<OwnerType> m_owner;
```

Comment:
I didn't check but this is almost certainly introducing a reference cycle and thus massive leaking. Replacing a raw C++ reference with a Ref<> is rarely a good idea. You likely want a WeakRef.

Reply:
Thanks for the tip!

Reply:
Yes this is correct. If the `OwnerType` is `SVGRectElement` and `SVGPropertyOwnerRegistry` owns `Ref<OwnerType>` then we will have this cyclic reference:

`Ref<SVGRectElement>` -> `UniqueRef<SVGPropertyRegistry>` -> `Ref<SVGRectElement>`